react/src/SearchVariants/components/SearchVariantsPage.tsx

```diff
@@ -124,7 +123,7 @@ const SearchVariantsPage: React.FC<SearchVariantsPageProps> = () => {
                         </Grid>
                         <Grid item>
                             <Button
-                                disabled={disableControls}
+                                // disabled={disableControls}
```

Comment:
This should stay un-commented. We don't want the user to have the option to download a report if they haven't specified what genes they want in their report.

Reply:
That makes sense! I'll uncomment - was leaving it there as a way to test the appearance of the snackbar.